xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/service/OkCoinAccountService.java

```diff
@@ -44,8 +50,34 @@ public String withdrawFunds(Currency currency, BigDecimal amount, String address
 
   @Override
   public String requestDepositAddress(Currency currency, String... args) throws IOException {
-
     throw new NotAvailableFromExchangeException();
   }
 
+  @Override
+  public FundsInfo getFundsInfo(TradeHistoryParams params) throws ExchangeException, NotAvailableFromExchangeException, NotYetImplementedForExchangeException, IOException{
+    final OkCoinFundsInfoHistoryParams histParams = (OkCoinFundsInfoHistoryParams) params;
+    final OkCoinAccountRecords depositRecord = getAccountRecords(histParams.getSymbol(), "0",
+            String.valueOf(histParams.getPageNumber() != null ? histParams.getPageNumber() : 1),
+            String.valueOf(histParams.getPageLength() != null ? histParams.getPageLength() : 1));
+    final OkCoinAccountRecords withdrawalRecord = getAccountRecords(histParams.getSymbol(), "1",
+            String.valueOf(histParams.getPageNumber() != null ? histParams.getPageNumber() : 1),
+            String.valueOf(histParams.getPageLength() != null ? histParams.getPageLength() : 1));
+    final OkCoinAccountRecords[] okCoinAccountRecordsList = new OkCoinAccountRecords[] {depositRecord, withdrawalRecord};
+    return OkCoinAdapters.adaptFundsInfo(okCoinAccountRecordsList);
+  }
+
+  public static class OkCoinFundsInfoHistoryParams extends DefaultTradeHistoryParamPaging {
+
+    private final String symbol;
```

Comment:
Dont [these methods](https://github.com/timmolter/XChange/blob/0bc7ff11e59c0244e6e805872ced024fe271d220/xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/OkCoinAdapters.java#L63) already solve this (perhaps with a bit of refactoring)?